projects/go-tester/src/app/app.component.ts

```diff
@@ -26,10 +29,38 @@ export class AppComponent implements OnInit {
   tableConfig: GoTableConfig;
   tableLoading: boolean = true;
 
+  menuItems: Array<NavGroup | NavItem> = [
```

Comment:
I don't think this is the right place to add our `menuItems`. Should we add somewhere else instead?

Reply:
@devchris this is 💯 for testing, this `go-tester` project will never go to production anywhere. This is our playground so I think it's fine.